src/usr/local/www/services_ntpd_gps.php

```diff
@@ -449,6 +483,13 @@ function build_nmea_list() {
 	base64_decode($pconfig['initcmd'])
 ))->setHelp('Commands entered here will be sent to the GPS during initialization. Please read and understand the GPS documentation before making any changes here.');
 
+$section->addInput(new Form_Checkbox(
+	'autocorrect_initcmd',
+	null,
+	'Auto correct malformed initialization commands. (default: checked).',
```

Comment:
The default values get set when switching GPS types, so if you have already set that then you will have to change the type to reset the default. I could change that behavior but that is how the other "defaults" are set.